composer/src/main/kotlin/com/gojuno/composer/Args.kt

```diff
@@ -9,32 +9,24 @@ data class Args(
         @Parameter(
                 names = arrayOf("--apk"),
                 required = true,
-                description = "Path to application apk that needs to be tested",
+                description = "Path to application apk that needs to be tested.",
                 order = 0
         )
         var appApkPath: String = "",
 
         @Parameter(
                 names = arrayOf("--test-apk"),
                 required = true,
-                description = "Path to apk with tests",
+                description = "Path to apk with tests.",
                 order = 1
         )
         var testApkPath: String = "",
 
-        @Parameter(
-                names = arrayOf("--test-package"),
-                required = true,
-                description = "Android package name of the test apk.",
-                order = 2
-        )
-        var testPackage: String = "",
-
         @Parameter(
```

Comment:
@artem-zinnatullin not sure if we still need that argument. I can imagine choosing from multiple runners in one test apk, though use cases are not clear. WDYT?

Reply:
Since it's now optional I'd leave it, but we need to update README and remove it from examples

Reply:
Yep, sure